src/io.rs

```diff
@@ -109,6 +115,7 @@ pub fn write_embeddings(
         FastText => bail!("Writing to the fastText format is not supported"),
         FinalFusion => embeddings.write_embeddings(&mut writer)?,
         FinalFusionMmap => bail!("Writing to memory-mapped finalfusion file is not supported"),
+        Floret => bail!("Writing to the floret format is not supported"),
```

Comment:
Because only floret vocabs can be serialized in foret format?

Reply:
Indeed. I think the handling of writing is a bit to coarse-grained. But I added this for now to avoid that people write incompatible embeddings in floret format.

Reply:
I still think this is a bit suboptimal in general. Because now you could use this code to read floret embeddings and then write them to e.g. word2vec, which would just give you an empty embedding matrix 😱.

I think this calls for some sort of compatibility function/table in `finalfusion-rust` that can tell use before doing any work whether a combination makes sense.